tool/src/webapp/js/announcements.js

```diff
@@ -0,0 +1,38 @@
+var number;
+$(function(){
+	number = $('.announcements-div').find($('#numberOfAnnouncements')).val();
+	//set the 'numberOfAnnouncements' in the hidden field for edit screen
+	$('#numberOfAnnouncements').val(number);
+	showAnnouncements();
+});
+
+function showAnnouncements(){
+	var url = $('.announcements-site-url').text().replace(/'/g,"");
+	var announcementsUrl = url + ".json?n=" + number;
+
+	//Get announcements
+	$.ajaxSetup({ cache: false,async: true });	//Disable caching for ajax requests but make asynchronous
+	$.getJSON(announcementsUrl, function(data, status, xhr) {
```

Comment:
What about displaying something when the `.getJSON` function fails for some reason?